pyfstat/grid_based_searches.py

```diff
@@ -722,6 +738,20 @@ def plot_2D(
         else:
             return ax
 
+    def get_max_det_stat(self):
+        """Get the maximum detection statistic over the grid.
+
+        This requires the `run()` method to have been called before.
+
+        Returns
+        -------
+        d: dict
+            Dictionary containing parameters and detection statistic at the maximum.
+        """
+        idx = np.argmax(self.data[self.detstat])
+        d = OrderedDict([(key, self.data[key][idx]) for key in self.output_keys])
```

Comment:
Do we need `OrderedDict`?  I though since 3.6 dictionaries preserved the order.

Reply:
Only since 3.7, apparently: https://docs.python.org/3.7/whatsnew/3.7.html

It also seems to retain a few extra methods, but we don't need those.

Reply:
Those willing to deprecate PyFstat's Python 3.6 support say `Yea` :drum: ...

Reply:
Let's make that a separate PR after lalsuite finally supports 3.9. I like having 3 supported versions. ;)